docs/api/spec/components/schemas/configcertificate.yaml

```diff
@@ -53,6 +59,9 @@ ConfigCertificateReturnValue:
 
 ConfigCertificateDetails:
   properties:
+    common_name:
+      type: string
+      description: "The common name (CN) of the certificate"
```

Comment:
Please use *The common name of the certificate* as you did elsewhere. Make sure to use the same string everywhere.